### DISCUSSION TITLE
GDPR Compliance

### DISCUSSION THREAD

**Discussion by bilboderbyshire** [pockethost/pockethost#244] [Q&A]
Is hosting a pocketbase instance on pockethost UK GDPR compliant?

I've searched and read through all the information available (I think) and haven't found anything explicitly saying that this is GDPR compliant.

I'm a teacher developing software for use in schools, for this to be possible it would need to be UK GDPR compliant. Is this something to check with the services pockethost uses (Amazon, DigitalOcean)?

I've probably missed/misunderstood something, my apologies if that's the case,

**Comment by benallfree** [comment]
@bilboderbyshire Is GDPR compliance something you’d like to tackle? I’m in the United States and there is no such requirement here.

**Reply by wderbyshire** [reply]
Hi, thanks for the reply,

As much as I'd love to help, I'm not sure how I would tackle this as I'm not a lawyer, nor that familiar with GDPR and how to make a service compliant. All services that are used in my school provide a statement claiming they (and all plugins and third-party services) are compliant. According to the regulation, GDPR compliance is a requirement for any entity that collects or processes user data of EU or UK residents. If PocketHost's plan is to be accessible by EU or UK citizens, it must become GDPR compliant.  It may simply be a case of identifying the services PocketHost uses to store its instances and ensuring they are GDPR compliant, but I can't be completely sure. I've provided a few links that I used to help me while developing:

- [https://www.termsfeed.com/blog/need-comply-gdpr/#:~:text=The%20GDPR%20states%20that%20any,be%20compliant%20with%20the%20GDPR.](url)
- [https://www.gov.uk/guidance/data-protection-in-schools/data-protection-policies-and-procedures](url)

My original question was just querying whether I would be legally covered to continue using PocketHost within my software as opposed to locally storing a PB instance on the school's network or using a generic hosting service like Linode.

Regardless, your service has been incredibly helpful while developing, and I greatly appreciate the awesome work you are doing with this.

**Reply by benallfree** [reply]
I just reviewed https://www.digitalocean.com/legal/gdpr-faq since PocketHost is a hosting provider. 

Based on what I see there, GDPR compliance seems well outside my expertise. In particular, compliance is difficult because there is no deterministic way to test compliance. 

That said, I’m sure we can take some reasonable steps over time to get into a position where we as a community reasonably believe that PocketHost is compliant. Off the top of my head:

* Privacy policy
* Data processing agreement like DO has for its customers: https://www.digitalocean.com/legal/data-processing-agreement
* Cookie controls
* Ability to permanently delete personal data

Also, each user needs to consider whether their use of PocketHost is compliant. PocketHost would not be expected to have knowledge of what personal information each customer might be collecting from *their* users. As I read the language, each service has an obligation to its direct users/customers only.

**Reply by wderbyshire** [reply]
Digital Ocean's statement should actually be enough to prove compliance since as you say, it would mainly be down to a PocketHost user to ensure their use and processing of user data is compliant. The angle my school was concerned about was whether the servers hosting the instances were adequately protected. A bad-case example they provided to me was having host servers based in China where security and access may not be trustworthy. However, since DO has a compliance statement and a DPR, that should be enough for my school.

I'll pass the information to them and see where they stand,
Many thanks for your help!
